docs/.vuepress/theme/components/Page.vue

```diff
@@ -51,6 +51,58 @@ export default {
     },
     isScriptLoaderActivated(exampleId) {
       return this.activatedExamples.includes(exampleId);
+    },
+    submitCodesandbox(submitEvent) {
+      // form data values
+      const html = submitEvent.target.elements.html.value;
+      const js = submitEvent.target.elements.js.value;
+      const css = submitEvent.target.elements.css.value;
+
+      const hyperformula = js.includes('hyperformula') ? { hyperformula: 'latest' } : {};
```

Comment:
@adrianspdev I think this part might be problematic when the documentation gets archived under a version number (when a newer version gets released).
It would be a better idea to use an actual Hyperformula version instead of `latest` here.

Reply:
for now we don't have easy way to do this, @evanSe we should create separate ticket for this case